ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -477,6 +477,17 @@ FAN_EFHLC_COCDSP_NOCDSP:
   - COCDSP
   - NOCDSP
 
+FAN_VSC_SCRSS:
+  description: "Kitchen exhaust fan with scrubber start/stop and status."
+  is_canonical: true
+  implements:
+  - FAN
+  - VSC
+  - SCRSS
+  opt_uses:
+  - bypass_air_damper_percentage_command
```

Comment:
do these belong on the scrubber? if so should be optional on that type, not the canonical.

Reply:
Updated in scrubber abstract.